includes/class-amp-reader-themes.php

```diff
@@ -0,0 +1,546 @@
+<?php
+/**
+ * Fetches and formats data for AMP reader themes.
+ *
+ * @package AMP
+ * @since 1.6.0
+ */
+
+/**
+ * Class AMP_Reader_Themes.
+ *
+ * @since 1.6.0
+ */
+final class AMP_Reader_Themes {
+	/**
+	 * Formatted theme data.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var array
+	 */
+	private $themes;
+
+	/**
+	 * The name of the currently active theme.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var string
+	 */
+	private $current_theme_name;
+
+	/**
+	 * Whether themes can be installed in the current WordPress installation.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var bool
+	 */
+	private $can_install_themes;
+
+	/**
+	 * The default reader theme. Indicates the plugin-provided classic mode should be used.
+	 *
+	 * @var string
+	 */
+	const DEFAULT_READER_THEME = 'classic';
+
+	/**
+	 * Status indicating a reader theme is active on the site.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var string
+	 */
+	const ACTIVE_STATUS = 'active';
+
+	/**
+	 * Status indicating a reader theme is installed but not active.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var string
+	 */
+	const INSTALLED_STATUS = 'installed';
```

Comment:
Same here regarding enum values:
```suggestion
	const STATUS_INSTALLED = 'installed';
```